person.rb

```diff
@@ -20,6 +21,11 @@ def can_use_services?
     (of_age? || @parent_permission)
   end
 
+  def add_rentals(rental)
+    rental << rental
+    rental.person = self
```

Comment:
Please create a new rental in `#add_rental` of `Person` class and pass person as a `self`, book, and date as parameters. You may see what parameters are needed by Rental's class constructor to create an object in order to pass the right params.